development/extensions/tutorial_php_events_and_extending_extensions.rst

```diff
@@ -284,10 +284,12 @@ implementation, your service configuration would look like this:
           - '@passwords.manager'
           - '@acme.demo.db_reader'
 
-Your PHP class should extend ``\phpbb\config\db`` which is original class in
-phpBB. To avoid any unforseen problems, you should not extend directly
-from ``\phpbb\config\config``. If both implemented the same interface, you could
-have extended from ``\phpbb\config\config``.
+The original config class in ``\phpbb\config\db`` didn't implement an interface.
```

Comment:
The original config class in ``\phpbb\config\db`` does not implement an interface.